src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.DocCommentFormatter.cs

```diff
@@ -200,11 +207,33 @@ internal static ImmutableArray<string> Format(IDocumentationCommentFormattingSer
                         firstInLine = true;
                     }
 
-                    current = end;
+                    span = span.Slice(end);
                 }
 
                 formattedLines.Add(sb.ToString());
             }
+
+            public static ImmutableArray<ReadOnlyMemory<char>> Split(
+                ReadOnlyMemory<char> s,
+                ReadOnlySpan<char> separator)
+            {
+                var result = ArrayBuilder<ReadOnlyMemory<char>>.GetInstance();
+
+                var index = s.Span.IndexOf(separator);
```

Comment:
Is this the same semantics as before?  What if the file only has \n newlines?  Before I believe it would still split ok.  But with this new logic, I'm not sure.

Reply:
Didn't the string.Split call take in a 1-element array, indicating "\r\n" is the delimiter? Was it intending to send in an array with '\r' and '\n' as delimiters instead?

Reply:
I'm not sure (and I'm on my phone). If the are the same semantics, great. It's just something I was nervous about. Thanks!

Reply:
It sure looks like the same semantics to me, but maybe I'm missing something as it does seem a little odd. (the Split call didn't change in your PR either, so if it's the same and it's wrong, it's been as such for a while)